### PR TITLE
Gave an example for Nabu Casa because many of the examples given else…

### PR DESCRIPTION
**Pull request by scstraus**
…where on github issues are wrong.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/withings.markdown [hunk 1]
```diff
@@ -25,7 +25,7 @@ Values for your account:
 - Logo: Any reasonable picture will do.
 - Description: Personal app for collecting my data.
 - Contact Email: Your email address
-- Callback Uri: `https://your-domain-name/auth/external/callback` - Withings will check if this URL is accessible (HTTP HEAD) upon submitting the form.
+- Callback Uri: `https://your-domain-name/auth/external/callback` - Withings will check if this URL is accessible (HTTP HEAD) upon submitting the form. If you are using a recent version of home assistant with Nabu Casa, this URL will look like `https://YOUR_NABU_CASA_ID.ui.nabu.casa/auth/external/callback`.
```
Comment (frenck) [inline source/_integrations/withings.markdown]:
```suggestion
- Callback Uri: `https://your-domain-name/auth/external/callback` - Withings will check if this URL is accessible (HTTP HEAD) upon submitting the form. If you are using the Home Assistant Cloud, this URL will look like `https://YOUR_NABU_CASA_ID.ui.nabu.casa/auth/external/callback`.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @scstraus!

Left a small comment, could you please take a look? Thanks! 👍

File: source/_integrations/withings.markdown (hunk 1 above)
Comment (scstraus) [reply source/_integrations/withings.markdown]: I’m okay with that.

**Review by frenck — APPROVED** [review]
Thanks, @scstraus 👍

**Review by scstraus — COMMENTED** [review]
Looks good.

**Pull request merged**